App/features/group/add-message/actions.ts

```diff
@@ -1,7 +1,20 @@
 import { createAsyncAction } from 'typesafe-actions'
 
+export interface addMessagePayload {
```

Comment:
Typically type/interface name are Capitalized. Maybe we should find a lint rule for that.